docs/registry.md

```diff
@@ -272,6 +272,20 @@ h1 registry transfer --registry test-registry --new-project OtherProject
 
 Manage your Repository of Registry
 
+### Examples
+
+#### Add tagged repository for registry
+
+```bash
+docker login 5d1090990136a7f0da513654.registry.pl-waw-1.hyperone.cloud -u login --password password
```

Comment:
Do not you want to explain where these values come from? You had a big problem with it yourself. See also: https://github.com/hyperonecom/h1-cli/search?q=Note&unscoped_q=Note